src/main/java/core/basesyntax/service/impl/CsvFileReader.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exception.CsvFileException;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.util.validator.PathValidator;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvFileReader implements FileReader {
+    private final Path source;
+
+    public CsvFileReader(Path source) {
```

Comment:
It's better to extract it from constructor and pass as a param to method, to make your service reusable for different destinations.